docs/Verilog.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title:  Introduction to Verilog Programming
+nav_order: 
+---
+
+# An Introduction to Verilog Programming
+{: .no_toc}
```

Comment:
Please check how other titles and Table Of Contents have been written. Please abide by the same.